ocaml_version.ml

```diff
@@ -234,6 +236,11 @@ module Has = struct
     match compare (Since.arch a) v with
     |(-1) | 0 -> true
     |_ -> false
+
+  let autoconf v =
+    match compare Since.autoconf v with
+    |(-1) | 0 -> true
```

Comment:
souldn't `Has.autoconf` too if the version is `> 4.08` instead of `<= 4.08`?

Reply:
That's what it does (this is consistent with other functions in `Has`):
`Has.autoconf v`
<=>
`compare Since.autoconf v <= 0`
<=>
`Since.autoconf <= v`
<=>
`v >= 4.08.0`